bin/plugin/commands/changelog.js

```diff
@@ -823,7 +823,17 @@ function getContributorPropsMarkdownList( ftcPRs ) {
  * @return {IssuesListForRepoResponseItem[]} The sorted list of pull requests.
  */
 function sortByUsername( items ) {
-	return sortBy( items, ( item ) => item.user.login.toLowerCase() );
+	return [ ...items ].sort( ( a, b ) => {
```

Comment:
Alternative to consider:

```js
return [ ...items ].sort( ( a, b ) => a.localeCompare( b, 'en', { sensitivity: 'accent' } ) );
```

You can also use `sensitivity: 'base'` if you want to ignore accents and other diacritics.

This comment also applies to all the other instances of string comparison.

Reply:
Hmm, I did consider using `localeCompare`, however, I didn't go that way because of having to specify a locale. But I'm happy to give it a try, considering that the locale setting is optional - it worked well in my tests. Thanks!

Reply:
Updated to use `localeCompare` in 370354f7b4ffd1bf53ca3ccadf7982a393c023c0. Could I ask you to take another look?

Reply:
Note that by not specifying a locale, you're making use of the system locale by default. Is that appropriate for all the scenarios here?

But more importantly than that, if you don't specify `sensitivity` it'll default to `variant`, which isn't case insensitive like you're looking for.

Reply:
> Note that by not specifying a locale, you're making use of the system locale by default. Is that appropriate for all the scenarios here?

It's more appropriate than guessing it's `en` of course. It's not like we maintain the current locale in the app anyway. That's why I didn't go with `localeCompare()` in the first place, TBH. Do you have any other suggestion to proceed here?

> But more importantly than that, if you don't specify sensitivity it'll default to variant, which isn't case insensitive like you're looking for.

Well, `sortBy()` is case sensitive as well - `_.sortBy( [ 'a', 'A', 'b', 'B' ] )` will yield `'A', 'B', 'a', 'b']`. So, we never actually wanted to sort in a case-insensitive way. If we want to do it case-insensitive, we specifically prepare the strings before sorting, like in this case we `.toLowerCase()` them (before the PR and in this PR). Or is it something else you're concerned about?

Reply:
> It's more appropriate than guessing it's en of course. It's not like we maintain the current locale in the app anyway.

The only reason I suggested `en` is because I guessed that stability was more important than absolute correctness for a given language; given what I know about the rules in English, I would expect `en` to be a good match for the previous algorithm. But to be fair, any other latin language locale would have worked reasonably well.

As for not going with `localeCompare()`, I'm not sure I follow. Are the rules for the previous algorithm more correct for this use-case? If so, then we should go back to it, but if not I don't think picking a particular language (or using the system default) is much of a concern.

In any case, I think using the system locale here is absolutely fine 👍

> like in this case we `.toLowerCase()` them (before the PR and in this PR)

No, sorry I wasn't too clear there; it was more a case of being able to drop `.toLowerCase()` instead of having to explicitly make that conversion.

Although if we are going with the system locale, we may want to consider `.toLocaleLowerCase()` as well for consistency 🤔

Reply:
Yup, happy with what's here atm 👍 Sorry if I over-complicated things 😅

Reply:
Actually, your feedback is much appreciated and there's always something to learn from it ❤️